utils/lcc-body-getter.js

```diff
@@ -0,0 +1,18 @@
+module.exports = (req) => {
```

Comment:
Here you can return a simple object
```suggestion
module.exports = {
    '1': 'lcc1',
    '2': 'lcc2',
    '3': 'lcc3'
}
```
And if you do this, I recommend placing in the files of the config directory instead of this file.